File: Java/reverse.java
import java.util.*;
public class reverse 
{
    public static void main(String[] args)
    {
        Scanner sc =new Scanner(System.in);
        System.out.println("Enter a number...:");
        int n=sc.nextInt();
        int rev=0;
        int p=n;
        int r;
        System.out.println("Original Number is: "+n);
      
        while(n>0)
        {
            r=n%10;
            rev=rev*10+r;
            n=n/10;
        }

        System.out.println("Reverse number is: "+rev);

        if(rev==p)
        {
            System.out.println(p+" is a palindrome number");
        }
        else
        {
            System.out.println(p+" is not an palindrome number");
        }
    }
}

File: c++/reverse.cpp
// Rverse of a number in cpp
#include <iostream>
#include<bits/stdc++.h>
using namespace std;
int main() {
    int n,rev=0,r;
        rev=rev*10+r;
    cout<<"Enter a number: "<<endl;
    cin>>n;
   
    while(n>0)
    {
        r=n%10;
        rev=rev*10+r;
        n=n/10;
    }
    cout<<"Reverse "<<" is"<<endl<<rev;

    return 0;
}
